IMPLEMENTATIONS.md

```diff
@@ -0,0 +1,12 @@
+# Implementations
+
+This is a list of projects which are using the all-contributors github bot. To add your project to this
+list, please [make a pull request](CONTRIBUTING.md)!
+
+- [all-contributors org](https://github.com/all-contributors)
+- [thechangelog org](https://github.com/thechangelog)
+- [netlify-cms](https://github.com/netlify/netlify-cms)
```

Comment:
Should this PR be merged after netlify/netlify-cms/issues/2015 is closed?

Reply:
@iamogbz FYI, https://github.com/netlify/netlify-cms/issues/2015 is in hold until #62 is resolved (according to [this comment](https://github.com/netlify/netlify-cms/issues/2015#issuecomment-458658942)).